pricegraph/src/api/price_source.rs

```diff
@@ -0,0 +1,89 @@
+//! This module implements price source methods for the `Pricegraph` API so that
+//! it can be used for OWL price estimates to the solver.
+
+use crate::encoding::{TokenId, TokenPair};
+use crate::{Pricegraph, FEE_TOKEN};
+
+const OWL_BASE_UNIT: f64 = 1_000_000_000_000_000_000.0;
+
+impl Pricegraph {
+    /// Estimates the fee token price in WEI for the specified token. Returns
```

Comment:
Maybe atoms instead of wei. Not sure what the standard ethereum term is but I thought wei was specifically for ether.